homework-1/src/tips.js

```diff
@@ -0,0 +1,64 @@
+const tipsPlusOne = document.querySelector('#tipsPlusOne');
```

Comment:
same as other js - bad solution. Find another one (a gave a link for accordion in comments above)

Reply:
done